zetaclient/chains/evm/signer/gas.go

```diff
@@ -92,7 +92,12 @@ func gasFromCCTX(cctx *types.CrossChainTx, logger zerolog.Logger) (Gas, error) {
 	case err != nil:
 		return Gas{}, errors.Wrap(err, "unable to parse priorityFee")
 	case gasPrice.Cmp(priorityFee) == -1:
```

Comment:
Is `gasFromCCTX` always the entrypoint for creating the gas object?
The current issue hit in the `Validate` function so we need to ensure Gas is always created with method

Reply:
I don't see any other code paths. @ws4charlie can you confirm?

Reply:
Yeah seems to be always created in `NewOutboundData` that use this function